### ISSUE TITLE
Curve not working on Raspberry Pi in QML

### ISSUE THREAD

**Issue by nO-OnE-910** [gre/bezier-easing#25]
I'm trying to use this project to ease animations in a QML canvas. Implementing and using works fine, at least when run on a Windows machine in Qt 5.7.0
But as soon as I try to run it cross compiled on a Raspberry Pi 3 the curve becomes a linear function. I have no idea what's going wrong here, Qt reports that JIT support is not enabled on the Pi, maybe that is a possible cause for the problem?
Every time calcBezier is run I print the input and output values, also float32ArraySupported. On both machines float32ArraySupported is true, and the values are about the same. The code is 100% identical.
I have no clue how to fix this.

My curve is (0.7, 0, 1-0.7, 1)

**Comment by gre** [comment]
can you show me a minimal test code that reproduce the bug?
thanks

**Comment by nO-OnE-910** [comment]
Okay, this is pretty stupid, and has nothing to do with your project. The operator that does not work on the Pi is ``` var int_v = real_v | 0 ```. It simply returns ```real_v``` while I was expecting it to return the integer part of the variable. On Windows this works fine.  
The way I call easing looks like ```var y = easing(real_v-int_v)+int_v```, which because the upper call returns ```real_v``` simply results in ```var y = real_v```. To me that looked like your function returned a linear curve.  
I'm sorry for this, thank you for your fast response though, since I eliminated all my variables from the call I was able to find the problem.
